pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "todo-app"
+version = "0.1.0"
+description = "A simple ToDo app for your DevOps apprenticeship"
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.7"
+flask = "^1.1.2"
+python-dotenv = "^0.14.0"
+requests = "^2.24.0"
+selenium = "^3.141.0"
+pytest = "^6.1.2"
```

Comment:
selenium and pytest should actually be in the `dev-dependencies` section below, so that you can skip them e.g. on the production server with `poetry install --no-dev`